x/cvm/simulation/sim_utils.go

```diff
@@ -103,6 +104,38 @@ func DeployContract(caller sim.Account, contractCode string, contractAbi string,
 	return msg, deployResponse.Result, nil
 }
 
+func RandomReasonableFees(r *rand.Rand, ctx sdk.Context, spendableCoins sdk.Coins) (sdk.Coins, error) {
```

Comment:
Does the simulation package not provide anything related to this?

Reply:
this implementation is almost copied from `RandomFees` at cosmossdk/types/simulation/account.go. The only modification that was made is to limit the randomized fees not exceeding ninth of the available balance.
There is `RandIntBetween` in sdk that could do a similar thing. But it's not handy to be used to generate Coins with random amounts.

Reply:
hmm sure. LGTM